src/super_gradients/common/data_connection/s3_connector.py

```diff
@@ -202,22 +189,21 @@ def download_key(self, target_path: str, key_to_download: str) -> bool:
             :return:                   True/False if the operation succeeded/failed
         """
         try:
-            self._logger.debug('Uploading Key: ' + key_to_download + ' from S3 bucket: ' + self.bucket_name)
+            self._logger.debug("Uploading Key: " + key_to_download + " from S3 bucket: " + self.bucket_name)
             self.s3_client.download_file(Bucket=self.bucket_name, Filename=target_path, Key=key_to_download)
         except botocore.exceptions.ClientError as ex:
-            if ex.response['Error']['Code'] == '404':
-                self._logger.error('[' + sys._getframe().f_code.co_name + '] - Key does exist in bucket)')
+            if ex.response["Error"]["Code"] == "404":
+                self._logger.error("[" + sys._getframe().f_code.co_name + "] - Key does exist in bucket)")
             else:
                 self._logger.critical(
-                    '[' + sys._getframe().f_code.co_name + '] - Caught Exception while trying to download key ' + str(
```

Comment:
here also please

Reply:
Fixed in 20ac8a290ce023b49df1cb9d67eb77c663a792d1